cmd/flipt/export.go

```diff
@@ -51,10 +52,17 @@ func newExportCommand() *cobra.Command {
 	)
 
 	cmd.Flags().StringVarP(
-		&export.namespace,
-		"namespace", "n",
+		&export.namespaces,
+		"namespaces", "n",
```

Comment:
i wonder if we can make this backward compatible by providing an alias for `namespace` and handle the single namespace case?

like here: https://github.com/GoogleContainerTools/kaniko/pull/1668/files#diff-dafa636d22231b15fa811fdcb798043e640b83c6a968fee91263d8305a406c93R59-R70

since these both use the short `n`  however this might be more difficult?

Reply:
@yquansah what if instead we keep `namespace` as is, but it only works for a single namespace and we mark it deprecated: https://pkg.go.dev/github.com/spf13/pflag#FlagSet.MarkDeprecated as well as the shorthand `n`: https://pkg.go.dev/github.com/spf13/pflag#FlagSet.MarkShorthandDeprecated

Then we add `namespaces` as a new flag with no shorthand. 

In a future release we can then remove the deprecated `namespace` flag and fully replace it with `namespaces`